src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -218,13 +218,117 @@ private void appendOneNativeArgument(ExecutionContext context, object obj, char
                         }
                         else
                         {
+#if UNIX
+                            // On UNIX systems, we expand arguments containing wildcard expressions against
+                            // the file system just like bash, etc.
```

Comment:
I have no recollection of any such thing. Why would we do that rather than using PowerShell's intrinsic globbing capability?